styles/styles.css

```diff
@@ -0,0 +1,153 @@
+@import url("./fonts.css");
+
+* {
+  margin: 0 auto;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  box-sizing: border-box;
+  max-width: 1440px;
+  height: 100vh;
+}
+
+.top-section {
+  width: 1440px;
+  height: 900px;
+  justify-content: space-between;
+  padding-top: 47px;
+  background: url(../images/castelmezzano-background-background.png) no-repeat center;
+  background-size: cover;
+  color: #ffffff;
+}
+
+.header {
+  display: flex;
+ }
+
+.logo-img {
+ background-image: url("../images/sprite.png");
+  width: 205px;
+  height: 50px;
+  margin-left: 104px;
+  margin-right: 583px;
+}
+
+.item-1 {
+  font-weight: 400;
+  font-size: 24px;
+  color: #ffffff;
+  margin-right: 24px;
+  text-decoration: none;
+  align-items: center;
+}
+
+.item-2 {
+  background-image: url("../images/sprite.png");
+  width: 30px;
+  background-position-x: -205px;
+  background-repeat: no-repeat;
+  margin-left: 24px;
+}
+
+.item-3 {
+  background-image: url("../images/sprite.png");
+  width: 40px;
+  background-position-x: -235px;
+  background-repeat: no-repeat;
+  margin-left: 24px;
+}
+
+.line {
+  border: 1px solid #f5bd41;
+  width: 50px;
+  margin-left: 998px;
+}
+
+.page {
+  padding-top: 200px;
+  padding-left: 105px;
+}
+
+.title-discover {
+  font-weight: 500;
+  font-size: 50px;
+  line-height: 59px;
+  margin-bottom: 98px;
+}
+
+.page-names {
+  display: flex;
+  font-weight: 400;
+  font-size: 18px;
+  text-align: left;
+}
+
+.names {
+  margin-left: 24px;
+  margin-right: 199px;
+}
+
+.form-search {
+  margin-top: 20px;
+}
+
+.form-city{
+  width: 463px;
+  height: 64px;
+  font-size: 18px;
+  border-bottom-left-radius: 8px;
+  border-top-left-radius: 8px;
+}
+
+.form-date{
+  width: 248px;
+  height: 64px;
+  font-size: 18px;
+  border-radius: 8px;
+  border: 4px solid #f5bd41;
+  text-align: center;
+}
+
+.form-room{
+  width: 305px;
+  height: 64px;
+  font-size: 18px;
+  text-align: center;
+}
+
+.form-submit{
+  width: 192px;
+  height: 71px;
+  font-size: 24px;
+  color: #ffffff;
+  text-align: center;
+  background-color: #3077C6;
+  border-radius: 8px;
+}
+
+.applications{
+  display: flex;
+  width: 21%;
+  margin-top: 55px;
+}
+
+.apps-1{
```

Comment:
Unclear name.

Reply:
Proper names from BEM I'm changing in 7th lesson